services/core-web/src/components/Forms/GeneratePermitForm.js

```diff
@@ -0,0 +1,229 @@
+          disabled={true}
+        />
+      </Col>
+      <Col xs={24} md={12}>
+        <Field
+          id="auth_end_date"
+          name="auth_end_date"
+          label="Authorization End Date"
+          required
+          component={renderConfig.DATE}
+          validate={[required]}
+        />
+      </Col>
+    </Row>
+    <Row gutter={32}>
+      <Col xs={24} md={12}>
+        <Field
+          id="lead_inspector"
+          name="lead_inspector"
+          label="Lead Inspector Name"
+          required
+          component={renderConfig.FIELD}
+          validate={[required]}
+          disabled={true}
+        />
+      </Col>
+      <Col xs={24} md={12}>
+        <Field
+          id="lead_inspector_title"
+          name="lead_inspector_title"
+          label="Lead Inspector Title"
+          required
+          component={renderConfig.FIELD}
+          validate={[required]}
+        />
+      </Col>
+    </Row>
+    <Row gutter={32}>
+      <Col xs={24} md={12}>
+        <Field
+          id="regional_office"
+          name="regional_office"
+          label="Regional Office"
+          required
+          component={renderConfig.FIELD}
+          validate={[required]}
+        />
+      </Col>
+      {props.isAmendment && (
+        <Col xs={24} md={12}>
+          <Field
+            id="original_permit_issue_date"
+            name="original_permit_issue_date"
+            label="Original Permit Issue Date"
+            required
+            component={renderConfig.DATE}
+            validate={[required]}
+          />
+        </Col>
+      )}
+    </Row>
+
+    <h2 className="padding-large--top">Part Two: Preamble</h2>
+    <Row gutter={32}>
+      <Col xs={24} md={12}>
+        <Field
+          id="application_date"
+          name="application_date"
+          label="Application Date"
+          required
+          component={renderConfig.FIELD}
+          validate={[required]}
+          disabled={true}
+        />
+      </Col>
+      <Col xs={24} md={12}>
+        <Field
+          id="application_type"
+          name="application_type"
+          label="Application Type"
+          required
+          component={renderConfig.FIELD}
+          validate={[required]}
+          disabled={true}
+        />
+      </Col>
+    </Row>
+
+    <h2 className="padding-large--top">Part Three: Permit Conditions</h2>
+    <Row gutter={32}>
+      <Col xs={24} md={12}>
+        <Field
+          id="conditions"
+          name="conditions"
+          label="Conditions"
+          required
+          component={renderConfig.AUTO_SIZE_FIELD}
+          validate={[required]}
+        />
+      </Col>
+    </Row>
+
+    <div className="right center-mobile">
+      <Popconfirm
+        placement="topRight"
+        title="Are you sure you want to cancel?"
+        okText="Yes"
+        cancelText="No"
+      >
+        <Button className="full-mobile" type="secondary" onClick={() => props.cancelGeneration()}>
```

Comment:
the onClick should be on the <Popconfirm > not the button, does this function as expected?


```
 <Popconfirm
          placement="topLeft"
          title={`Are you sure you want to delete`}
          onConfirm={() => props.delete()}
          okText="Delete"
          cancelText="Cancel"
        >
          <Button>
            Cancel
          </Button>
        </Popconfirm>
```

Reply:
changed